test/smoke/testApps/OpenTelemetryMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/OpenTelemetryMetricTest.java

```diff
@@ -104,6 +104,8 @@ private void validateHistogramMetric(String name) throws Exception {
     assertEquals(456, dp.getValue(), 0);
     assertEquals(DataPointType.Aggregation, dp.getKind());
     assertEquals(name, dp.getName());
+    assertNotNull(dp.getMin());
+    assertNotNull(dp.getMax());
```

Comment:
can we assertEquals 456 for both of these?

also, looks like we are missing assertion on count

Reply:
why is 456? source please.